### DISCUSSION TITLE
Multi-instance learning tutorial Loss

### DISCUSSION THREAD

**Discussion by yarkinslav** [Project-MONAI/tutorials#1478] [Q&A]
Hello, thanks for great tutorial for multi-instance learning. I am trying to understand notation (3) from the paper L = Lbag + lambda * Lpatch. From the tutorial we have [nn.BCEWithLogitsLoss()](https://github.com/Project-MONAI/tutorials/blob/12914febe668b079bdef983da5b5b4d8839e6e57/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py#L52), with "mean" reduction with predefined weight decay. I could understand that we are able to use use BCE loss because of [one-hot label encoding map transform](https://github.com/Project-MONAI/tutorials/blob/12914febe668b079bdef983da5b5b4d8839e6e57/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py#L200C7-L200C31) , so we assign each prediction to 1 class. But I can't get underlying connection notation (3) with the defined BCELosswithLogits. Can someone elaborate on this?

**Comment by KumoLiu** [comment]
Hi @relyativist, thanks for your interest here.
The tutorial was not written exactly according to the paper, and it did not include the process of generating the pseudo label. So IMO, it doesn't match the notation (3). We only contained bag-level loss.

If @myron could help double-confirm, that would be great! Thanks in advance!

**Reply by yarkinslav** [reply]
@KumoLiu Thanks for your reply! Would be great to get the details!
